microsoft-dotnet/3.1/modules/ROOT/pages/microsoft-dotnet-connector-examples.adoc

```diff
@@ -0,0 +1,119 @@
+= Microsoft .NET Connector 3.1 Examples - Mule 4
+
+The following use case...
+*Reviewers - please summarize what this use case does and also look at the example carefully. I added more detailed steps.*
+
+image::microsoft-dotnet-use-case.png[Studio Flow Diagram]
+
+. Create a new Mule project in Studio.
+. In the *Mule Palette* view, search for `http` and select the *Listener* operation.
+. Drag the *Listener* operation onto the Studio canvas.
+. In the *Listener* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. Set the *Host* field to `localhost`.
+. Set the *Path* field to `sumOfDigits`.
+. From the *Mule palette* view, drag a *Transform Message* component to right of *Listener*.
+. Click *Transform Message* and set the output as follows *Reviewers, what does this code transform?*:
++
+[source,dataweave,linenums]
+----
+dw 2.0
+output application/json
+---
+{number: payload.num
+}
+----
++
+. In the *Mule Palette* view, search for `dotnet` and drag the *Microsoft DotNet* > *Execute* operation onto the Studio canvas, to the right of *Transform Message*.
+. In the *Execute* configuration, click *+* next to the *Connector configuration* field to add a global element.
+. In the *Scope* field, select `Singleton`.
+. Click *OK*.
```

Comment:
before clicking `OK` button there is one more step:
Go to `Assembly` tab and add the `DemoDotNet.dll` in `Path` field.